app/account/error.tsx

```diff
@@ -0,0 +1,50 @@
+'use client'
```

Comment:
I actually added error page in app directory, so you don't need this page!
`/Tabimemo/frontend/app/error.tsx`